pylintrc_backup

```diff
@@ -0,0 +1,461 @@
+# ***************************
```

Comment:
We shouldn't need to push this file. It's only created as backup when running `edx_lint write pylintrc` command.